path/src/f32x2_t.rs

```diff
@@ -104,3 +104,11 @@ impl core::ops::Div<f32x2> for f32x2 {
         ])
     }
 }
+
+fn pmax(a: f32, b: f32) -> f32 {
```

Comment:
@CryZe Is this a no_std variant?

Reply:
No, basically as I've noticed back then when I opened this PR, there's a bunch of platform differences when it comes to NaN, but Skia doesn't seem to care about them. Rust's default max is "unnecessarily slow" in that it cares for NaN (and I believe other edge cases), so I ported the pmin/pmax logic from wasm (that also is meant to ignore NaN and just do the fastest thing possible) so it can be used in the fallback code.

Reply:
Are you sure? It seems it's just an intrinsic: ihttps://doc.rust-lang.org/stable/src/core/num/f32.rs.html#749

Reply:
I compiled this list back then that really shows how much of a mess min and max are: https://gist.github.com/CryZe/30cc76f4629cb0846d5a9b8d13144649

Reply:
Yes, it just calls into fmaxf, which is C's overcomplicated function: https://rust.godbolt.org/z/KKz7875sr

Reply:
Ok, I will look into it in more details. We can leave it as is for now.

Reply:
https://docs.rs/libm/latest/src/libm/math/fmaxf.rs.html#11

Reply:
Hmm, I see. We technically shouldn't have NaN values to begin with, so a custom version should be fine.